src/plugins/ui_stateful/sheetview.ts

```diff
@@ -316,6 +317,15 @@ export class SheetViewPlugin extends UIPlugin {
     };
   }
 
+  getVisibleSheetViewRect(): Rect {
```

Comment:
Are we ok with getters in o_spreadsheet that are only used in Odoo ? We could add this getter to a random odoo plugin

Reply:
this exposes the internal state of the plugin, so yeah I could use existing getters to infer it, it just looked super shitty.

Reply:
Yeah both solution sucks, I was just wondering our stance on "getter only used in odoo" vs "wonky getter only in odoo"